components/emoji_picker/emoji_picker.jsx

```diff
@@ -112,6 +112,10 @@ function getEmojiFilename(emoji) {
     return emoji.filename || emoji.id;
 }
 
+export function filterEmojiSearchInput(input) {
+    return input.toLowerCase().replace(/^:/g, '').replace(/:$/g, '');
```

Comment:
I think the regex can be combined, something like,
```suggestion
    return input.toLowerCase().replace(/^:|:$/g, '');
```